pkg/config/config.go

```diff
@@ -162,9 +162,12 @@ func Load(scheme *runtime.Scheme, configFile string) (ctrl.Options, configapi.Co
 			return options, cfg, err
 		}
 	}
-	if err = validate(&cfg).ToAggregate(); err != nil {
-		return options, cfg, err
+
+	errs := ValidateConfiguration(cfg)
+	if ea := errs.ToAggregate(); ea != nil {
```

Comment:
nit: IIUC the changes aren't necessary, I would suggest to avoid them.

Reply:
Reverted the changes